Unable to instantiate receiver

<!--
DELETE ME

Use the search function to see if someone else has already submitted the same bug report.

You don't need to adhere to the template strictly. Feel free to leave out information you feel is not important or does not make sense.

If you are experiencing a crash, including the stacktrace will likely get it fixed sooner.
-->

**App version**: develop

Probably caused by the switch to WorkManager. Happens once after reboot.

**Stacktrace/Logcat**: 
```
java.lang.RuntimeException: Unable to instantiate receiver de.danoeh.antennapod.core.receiver.AlarmUpdateReceiver: java.lang.ClassNotFoundException: Didn't find class "de.danoeh.antennapod.core.receiver.AlarmUpdateReceiver" on path: DexPathList[[zip file "/system/framework/org.apache.http.legacy.boot.jar", zip file "/data/app/de.danoeh.antennapod-yFR9niKpqBAIseqNTZStsA==/base.apk"],nativeLibraryDirectories=[/data/app/de.danoeh.antennapod-yFR9niKpqBAIseqNTZStsA==/lib/arm64, /system/lib64, /system/vendor/lib64]]
	at android.app.ActivityThread.handleReceiver(ActivityThread.java:3403)
	at android.app.ActivityThread.access$1200(ActivityThread.java:200)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1667)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:193)
	at android.app.ActivityThread.main(ActivityThread.java:6718)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
Caused by: java.lang.ClassNotFoundException: Didn't find class "de.danoeh.antennapod.core.receiver.AlarmUpdateReceiver" on path: DexPathList[[zip file "/system/framework/org.apache.http.legacy.boot.jar", zip file "/data/app/de.danoeh.antennapod-yFR9niKpqBAIseqNTZStsA==/base.apk"],nativeLibraryDirectories=[/data/app/de.danoeh.antennapod-yFR9niKpqBAIseqNTZStsA==/lib/arm64, /system/lib64, /system/vendor/lib64]]
	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:134)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
	at android.app.AppComponentFactory.instantiateReceiver(AppComponentFactory.java:84)
	at android.app.ActivityThread.handleReceiver(ActivityThread.java:3396)
	... 8 more

```

--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -366,18 +366,6 @@
                 <action android:name="de.danoeh.antennapdsp.intent.SP_APPS_QUERY_FEEDS_RESPONSE"/>
             </intent-filter>
         </receiver>
-        <receiver android:name="de.danoeh.antennapod.core.receiver.AlarmUpdateReceiver">
-            <intent-filter>
-                <action android:name="android.intent.action.BOOT_COMPLETED"/>
-            </intent-filter>
-            <intent-filter>
-                <action android:name="android.intent.action.PACKAGE_REPLACED"/>
-
-                <data
-                    android:path="de.danoeh.antennapod"
-                    android:scheme="package"/>
-            </intent-filter>
-        </receiver>
         
         <provider
             android:authorities="@string/provider_authority"

